### PR TITLE
feat: staking free

### PR DESCRIPTION
**Pull request by Jakub123Jakub123**

### CODE REVIEW

File: contrib/build-wine/build-electrum-git.sh [hunk 1]
```diff
@@ -26,52 +18,57 @@ info "Last commit: $VERSION"
...
-find -exec touch -d '2000-11-11T11:11:11+00:00' {} +
+find -exec touch -h -d '2000-11-11T11:11:11+00:00' {} +
 popd
 
 info "building NSIS installer"
 # $VERSION could be passed to the electrum.nsi script, but this would require some rewriting in the script itself.
 wine "$WINEPREFIX/drive_c/Program Files (x86)/NSIS/makensis.exe" /DPRODUCT_VERSION=$VERSION electrum.nsi
 
 cd dist
-mv elcash-wallet-setup.exe $NAME_ROOT-$VERSION-setup.exe
```
Comment (krecik) [inline contrib/build-wine/build-electrum-git.sh:74]: Should be elcash-wallet

File: contrib/build-wine/build-electrum-git.sh [hunk 2]
```diff
@@ -116,4 +113,4 @@ EOF
     done
 )
 
-sha256sum dist/elcash-wallet*.exe
```
Comment (krecik) [inline contrib/build-wine/build-electrum-git.sh:119]: elcash-wallet

File: contrib/build-wine/electrum.nsi [hunk 3]
```diff
@@ -2,12 +2,12 @@
 ;Include Modern UI
   !include "TextFunc.nsh" ;Needed for the $GetSize function. I know, doesn't sound logical, it isn't.
   !include "MUI2.nsh"
-  
+
 ;--------------------------------
 ;Variables
 
-  !define PRODUCT_NAME "ELCASH Wallet"
```
Comment (krecik) [inline contrib/build-wine/electrum.nsi:9]: ELCASH Wallet

File: contrib/build-wine/electrum.nsi [hunk 4]
```diff
@@ -2,12 +2,12 @@
...
-  !define PRODUCT_WEB_SITE " https://electriccash.global/"
```
Comment (krecik) [inline contrib/build-wine/electrum.nsi:10]: Proper website address.

File: contrib/build-wine/electrum.nsi [hunk 5]
```diff
@@ -16,7 +16,7 @@
 
   ;Name and file
   Name "${PRODUCT_NAME}"
-  OutFile "dist/elcash-wallet-setup.exe"
```
Comment (krecik) [inline contrib/build-wine/electrum.nsi:19]: Should be `elcash-wallet-setup.exe`

**Review by krecik — CHANGES_REQUESTED** [review]

File: contrib/build-wine/build-electrum-git.sh [hunk 6]
```diff
@@ -1,21 +1,13 @@
 #!/bin/bash
 
-NAME_ROOT=elcash-wallet
+NAME_ROOT=electrum
```
Comment (krecik) [inline contrib/build-wine/build-electrum-git.sh]: elcash-wallet

File: contrib/build-wine/build-electrum-git.sh [hunk 7]
```diff
@@ -26,52 +18,57 @@ info "Last commit: $VERSION"
...
+find -exec touch -h -d '2000-11-11T11:11:11+00:00' {} +
 popd
 
 info "building NSIS installer"
 # $VERSION could be passed to the electrum.nsi script, but this would require some rewriting in the script itself.
 wine "$WINEPREFIX/drive_c/Program Files (x86)/NSIS/makensis.exe" /DPRODUCT_VERSION=$VERSION electrum.nsi
 
 cd dist
-mv elcash-wallet-setup.exe $NAME_ROOT-$VERSION-setup.exe
+mv electrum-setup.exe $NAME_ROOT-$VERSION-setup.exe
```
Comment (krecik) [inline contrib/build-wine/build-electrum-git.sh]: `elcash-wallet-setup.exe`

File: contrib/build-wine/build-electrum-git.sh [hunk 8]
```diff
@@ -116,4 +113,4 @@ EOF
     done
 )
 
-sha256sum dist/elcash-wallet*.exe
+sha256sum dist/electrum*.exe
```
Comment (krecik) [inline contrib/build-wine/build-electrum-git.sh]: `elcash-wallet*.exe`

File: contrib/build-wine/electrum.nsi [hunk 9]
```diff
@@ -16,7 +16,7 @@
...
+  OutFile "dist/electrum-setup.exe"
```
Comment (krecik) [inline contrib/build-wine/electrum.nsi]: "dist/elcash-wallet-setup.exe"

**Review by krecik — DISMISSED** [review]

**Review by diabelko — APPROVED** [review]

**Pull request merged**